pkg/ccl/changefeedccl/sink_cloudstorage.go

```diff
@@ -312,6 +312,12 @@ func makeCloudStorageSink(
 	makeExternalStorageFromURI cloud.ExternalStorageFromURIFactory,
 	user security.SQLUsername,
 ) (Sink, error) {
+	// cloud storage sink has no sink-specific options
+	err := validateSinkOptions(opts, nil)
```

Comment:
Let's add a variable (= nil) for all sink types so that everything is consistent.

Reply:
added cloudstorage (which is non-nil now) and SQL.